.github/workflows/merge_checks.yml

```diff
@@ -17,16 +17,27 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - name: Style check with py310-ubuntu
-            python: '3.10'
+          - name: Style check with py311-ubuntu
+            python: '3.11'
             arch: x64
             os: ubuntu-latest
             toxenv: style
+          # Type checking fails for py39, don't know how to resolve
```

Comment:
so how does this work, would a python 3.9 project using this package also automatically not longer have passing tests? Or is it only an internal type check that fails (and cannot be simply ignored?)

Reply:
The problem occurs for method _get, which is private. So client code should not have a problem. The annoying thing is that I have not found a way to suppress the error... I got tired of it and decided to not do the type checking for 3.9 (we did not do it before either)

Reply:
There is also the possibility of a local ignore of typing errors:
https://mypy.readthedocs.io/en/stable/error_codes.html#silencing-errors-based-on-error-codes

Or would that then litter the code base at every place where we call `_get`?

Although the downside is then of course that we silence it for all python versions... Probably this is indeed fine for now, and just revisit it later.

Reply:
I have been playing around with that, but not yet a satisfactory result. Will look at it some more tomorrow.

Reply:
Not needed anymore.